fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -2990,10 +2991,15 @@ private boolean isEnableFillDataCacheImpl(Partition partition) throws AnalysisEx
                 dataCacheRange = Range.openClosed(PartitionKey.ofDateTime(lower), PartitionKey.ofDateTime(upper));
                 return partitionRange.isConnected(dataCacheRange);
             } else if (rangePartitionInfo.isPartitionedBy(PrimitiveType.DATE)) {
-                LocalDate upper = LocalDate.now();
-                LocalDate lower = upper.minus(cacheDuration);
-                dataCacheRange = Range.openClosed(PartitionKey.ofDate(lower), PartitionKey.ofDate(upper));
-                return partitionRange.isConnected(dataCacheRange);
+                try {
+                    LocalDate upper = LocalDate.now();
+                    LocalDate lower = upper.minus(cacheDuration);
+                    dataCacheRange = Range.openClosed(PartitionKey.ofDate(lower), PartitionKey.ofDate(upper));
+                    return partitionRange.isConnected(dataCacheRange);
+                } catch (UnsupportedTemporalTypeException e) {
+                    LOG.warn("{}. table name: {}, datacache.partition_duration: {}, mismatch with partition range.",
```

Comment:
so it will return `true` anyway?

need add UT to cover this `isEnableFillDataCache()` interface.